Modules/System Tests/GeoLocation/src/GeoLocationTest.Codeunit.al

```diff
@@ -0,0 +1,153 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 50104 "GeoLocation Test"
+{
+    // [FEATURE] [GeoLocation] 
+
+    Subtype = Test;
+    TestPermissions = Disabled;
+
+    var
+        Assert: Codeunit "Library Assert";
+
+    [Test]
+    [Scope('OnPrem')]
+    procedure TestIsGeoLocationAvailable()
+    var
+        GeoLocation: Page GeoLocation;
+        GeoLocationTestLibrary: Codeunit "GeoLocation Test Library";
+    begin
+        // [When] GeoLocation test library subscribers are binded.
+        BindSubscription(GeoLocationTestLibrary);
+
+        // [Then] The return value of IsAvailable is 'true'.
+        Assert.IsTrue(GeoLocation.IsAvailable(), 'The GeoLocation should be available during testing.');
```

Comment:
The message that provided is shown when the assert fails. So the message should be the opposite.
Something like 'The GeoLocation is unavailable'.